dapps/valuetransfers/dapp.go

```diff
@@ -67,15 +69,41 @@ var (
 	valueObjectFactoryOnce sync.Once
 )
 
+// App gets the plugin instance
+func App() *node.Plugin {
+	appOnce.Do(func () {
+		app = node.NewPlugin(PluginName, node.Enabled, configure, run)
+	})
+	return app
+}
+
+// Tangle gets the tangle instance
```

Comment:
add here the original comment from line 52. Would be nice to have a descriptive comment for all exported functions, so my editor tells me exactly what a `valuetransfers.Tangle()` is when I want to use it from outside. Same goes for all the newly introduced singleton getters  in this PR below.